tests/acceptance/features/bootstrap/WebUIUsersContext.php

```diff
@@ -563,8 +580,21 @@ public function restoreScenario(AfterScenarioScope $afterScenarioScope) {
 	 */
 	public function theAdminAddsGroupUsingTheWebUI($groupName) {
 		$this->usersPage->addGroup($groupName, $this->getSession());
+		\array_push($this->groupsCreated, $groupName);
 	}
 
+	/**
+	 * @AfterScenario
+	 *
+	 * @return void
+	 * @throws \Exception
+	 */
+	public function cleanupGroups() {
```

Comment:
let the featureContext deal with the cleanup https://github.com/owncloud/core/blob/5fff7599545af9e365eda39af3c52798f0517208/tests/acceptance/features/bootstrap/Provisioning.php#L2860
and even if you have to do it here, do not use the UI to clean-up, the API is much faster